DEFINITION.md

```diff
@@ -2,7 +2,7 @@
 
 *Approved by TOC: 2018-06-11*
 
-[中文版本](#中文版本) | [日本語版](#日本語版) | [한국어](#한국어) | [Português Brasileiro](#português-brasileiro) (in Chinese, Japanese, Korean and Brazilian Portuguese below)
+[中文版本](#中文版本) | [日本語版](#日本語版) | [한국어](#한국어) | [Português Brasileiro](#português-brasileiro) | [German](#german) (in Chinese, Japanese, Korean and Brazilian Portuguese below)
```

Comment:
`Deutsch` seems better :) sorry for not bring it up in the first place!

Reply:
you are right